app/assets/javascripts/spritegroup.js

```diff
@@ -0,0 +1,31 @@
+function SpriteGroup (spriteName, spriteGroup, objectPool, numSprite, spriteOriginPosX, spriteOriginPosY) {
+  this.spriteName = spriteName;
+  this.spriteGroup = spriteGroup;
+  this.numSprite = numSprite;
+  this.spriteOriginPosX = spriteOriginPosX;
+  this.spriteOriginPosY = spriteOriginPosY;
+
+  this.renderSprites = function (shape) {
+    var spriteFileName = "";
+    var spritePosX = spriteOriginPosX;
+    var spritePosY = spriteOriginPosY;
+
+    // todo: renders in an octagonal fashion
+
+    // renders in a box like fashion
+    if (shape == "box") {
+      console.log(objectPool.allObjects.length);
+      for (var index in objectPool.allObjects) {
+        if (objectPool.allObjects[index].id > 0) {
+          spriteFileName = spriteName + objectPool.allObjects[index].id;
+          spriteGroup.add(game.add.sprite(spritePosX, spritePosY, spriteFileName));
```

Comment:
Adds the sprite to the group defined by Phaser.group, see: http://phaser.io/docs/2.4.4/Phaser.Group.html